packages/data-context/src/data/ProjectLifecycleManager.ts

```diff
@@ -453,6 +453,8 @@ export class ProjectLifecycleManager {
         }
       })
 
+      fullConfig.resolved.browsers = { 'value': fullConfig.browsers, 'from': 'runtime' }
```

Comment:
I see the conditional above, `if (!fullConfig.browsers || fullConfig.browsers.length === 0) {`... should we add 
```
fullConfig.browsers = browsers
fullConfig.resolved.browsers = { 'value': fullConfig.browsers, 'from': 'config' }
```

and then only add the `from: 'runtime'` if we haven't added `fullConfig.resolved.browsers`?